dipy/workflows/base.py

```diff
@@ -45,12 +45,12 @@ def __init__(self, prog=None, usage=None, description=None, epilog=None,
 
         iap = IntrospectiveArgumentParser
         if epilog is None:
-            epilog = \
-                "References: \n" \
-                "Garyfallidis, E., M. Brett, B. Amirbekian, A. Rokem," \
-                " S. Van Der Walt, M. Descoteaux, and I. Nimmo-Smith. Dipy, a" \
-                " library for the analysis of diffusion MRI data. Frontiers " \
-                "in Neuroinformatics, 1-18, 2014."
+            epilog =\
+                ("References: \n"
+                 "Garyfallidis, E., M. Brett, B. Amirbekian, A. Rokem,"
+                 " S. Van Der Walt, M. Descoteaux, and I. Nimmo-Smith. Dipy, a"
+                 " library for the analysis of diffusion MRI data. Frontiers"
+                 " in Neuroinformatics, 1-18, 2014.")
```

Comment:
I definitly wouldnt mind usig it. I suggest making a separate PR for this afterwards.